### ISSUE TITLE
RE 2, room 21A, crash potentially caused by moths

### ISSUE THREAD

**Issue by baesik** [IntelOrca/biorand#360]
### Version

3.0.1 (b5e7489)

### Seed

R721-8NZ0-ZTZZ00NK9W0A0L49K499ZZZZZZTZZZZZF3Z

### Scenario

RE 2 (Claire A)

### Player Character

Annette (RE2R)

### What happened?

I believe the cause to be the moths because I have encountered lickers, dogs, tyrant, and crows in this room and it has never crashed on me before. 

~~The room itself is kind of problematic. The cell with the blue herb is a bit wonky. For the blue herb to appear, you must enter the cell from Cut 4. Entering via any other cut won't show the item on the floor.~~
Edit: I'm dumb. After years and years of playing I now see three cells, not two. 😵‍💫

Save data and video are attached.
[Claire A_00_2023-05-13 18^42.42.zip](https://github.com/IntelOrca/biorand/files/11470942/Claire.A_00_2023-05-13.18.42.42.zip)

https://github.com/IntelOrca/biorand/assets/16945591/b1eb9963-cba3-4d94-acf6-0760964df13f

**Comment by Soullesscat** [comment]
When I reached the same room I had different enemies due to randomization but judging from my own experience moths can jump out of bounds so it's possible that this causes a crash. 

![image](https://github.com/IntelOrca/biorand/assets/36854965/acc6000e-e784-48b0-969f-4a160574d376)

I saw it in a different room which was the room where Mr.X grabs the railing and jumps onto the most upper level of the main hall next to the library and the cogwheel puzzle. The moths managed to jump from the upper level down into the main hall and X has to wait somewhere out of bounds too or needs extra space to grab onto the railing so maybe it was not a problem for an enemy to go out of bounds there unlike in room 21A.

![image](https://github.com/IntelOrca/biorand/assets/36854965/34002fcb-96bb-4f57-b6e2-5f4a68d09bea)
![image](https://github.com/IntelOrca/biorand/assets/36854965/40e70ed1-106b-49f9-9f60-da2ef63d2218)
